srl/src/main/java/edu/illinois/cs/cogcomp/srl/utilities/WeightVectorUtils.java

```diff
@@ -0,0 +1,134 @@
+/**
```

Comment:
oh I see... but the `core-utilities` dependencies are limited to just `IOUtils` and `illinois-sl` already has a solution for these methods (see [WeightVector](https://github.com/IllinoisCogComp/illinois-sl/blob/master/src/main/java/edu/illinois/cs/cogcomp/sl/util/WeightVector.java)). So maybe just a quick fix and this file will be ready to be moved to `sl`